elephant/spike_train_correlation.py

```diff
@@ -749,3 +751,68 @@ def run_T(spiketrain):
 
 
 sttc = spike_time_tiling_coefficient
+
+
+def spike_train_timescale(binned_st, tau_max):
+    """
+    Calculates the auto-correlation time of a binned spike train.
+    Uses the definition of the auto-correlation time proposed in [1, Eq. (6)]:
+
+    .. math::
+        \\tau_\mathrm{corr} = \\int_{-\\tau_\mathrm{max}}^{\\tau_\mathrm{max}}\
+            \left[ \\frac{\hat{C}(\\tau)}{\hat{C}(0)} \\right]^2 d\\tau
+
+    where :math:`\hat{C}(\\tau) = C(\\tau)-\\nu\delta(\\tau)` denotes
+    the auto-correlation function excluding the Dirac delta at zero timelag.
+
+    Parameters
+    ----------
+    binned_st : elephant.conversion.BinnedSpikeTrain
+        A binned spike train containing the spike train to be evaluated.
+    tau_max : quantities.Quantity
+        Maximal integration time of the auto-correlation function.
+
+    Returns
+    -------
+    timescale : quantities.Quantity
+        The auto-correlation time of the binned spiketrain.
+
+    Notes
+    -----
+    * :math:`\\tau_\mathrm{max}` is a critical parameter: numerical estimates
+      of the auto-correlation functions are inherently noisy. Due to the
+      square in the definition above, this noise is integrated. Thus, it is
+      necessary to introduce a cutoff for the numerical integration - this
+      cutoff should be neither smaller than the true auto-correlation time
+      nor much bigger.
+    * The binsize of binned_st is another critical parameter as it defines the
+      discretisation of the integral :math:`d\\tau`. If it is too big, the
+      numerical approximation of the integral is inaccurate.
+
+    References
+    ----------
+    [1] Wieland, S., Bernardi, D., Schwalger, T., & Lindner, B. (2015).
```

Comment:
It is in the paper, Eq. (6) - as is written in the docstring (line 759). Here is the doi in case you looked at the wrong paper: 10.1103/PhysRevE.92.040901

Reply:
ah, true, i missed it!

Reply:
no prob